src/components/Header/web3/NetworkModal/DraggableNetworkButton.tsx

```diff
@@ -162,10 +162,10 @@ export default function DraggableNetworkButton({
       customOnSelectNetwork(chainId)
     } else if (getChainType(chainId) === getChainType(chainId)) {
       changeNetwork(chainId, () => {
-        const { inputCurrency, outputCurrency, ...rest } = qs
+        // const { inputCurrency, outputCurrency, ...rest } = qs
```

Comment:
why sir

Reply:
I need to keep param since we have omni view, then chainId from swap form is persisted 
<img width="414" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/11427213/6c56d842-9060-4f66-ab16-d96f7e4724d1">